.github/workflows/k8s-snap-integration.yaml

```
@@ -59,7 +59,11 @@ jobs:
           TEST_LXD_IMAGE: ubuntu:22.04
           TEST_INSPECTION_REPORTS_DIR: ${{ github.workspace }}/inspection-reports
         run: |
-          git clone https://github.com/canonical/k8s-snap.git      
+          if [[ "${{ contains(github.event.pull_request.labels.*.name, 'tag-that-Louise-choose') || github.ref_name  == 'master' }}" == "true" ]] ; then
```

instead can be. idk which is better from maintenance point of view 
```suggestion
          if  "${{ contains(github.event.pull_request.labels.*.name, 'tag-that-Louise-choose') || github.ref_name  == 'master' }}"; then
```